Geometry/HGCalGeometry/src/HGCalGeometry.cc

```diff
@@ -233,7 +237,7 @@ GlobalPoint HGCalGeometry::getPosition(const DetId& detid, bool debug) const {
                                         << id.iCell2;
       }
       xy = m_topology.dddConstants().locateCell(
-          id.zSide, id.iLay, id.iSec1, id.iSec2, id.iCell1, id.iCell2, true, true, false, debug);
+          id.zSide, id.iLay, id.iSec1, id.iSec2, id.iCell1, id.iCell2, true, true, true, cog, debug);
```

Comment:
@bsunanda was it intended to flip the `norot` value from `false` to `true`, while adding the `cog` parameter ?

Reply:
Unfortunately the summary and comparisons of the test are not there any more. 
@bsunanda , did you have a look at the 4047 differences found?